pkg/virt-handler/vm.go

```diff
@@ -99,7 +99,9 @@ func (d *VMHandlerDispatch) Execute(store cache.Store, queue workqueue.RateLimit
 	if !exists {
 		// Since the VM was not in the cache, we delete it
 		err = d.domainManager.KillVM(vm)
+		logging.DefaultLogger().V(3).Info().Object(vm).Msg("Killing VM.")
```

Comment:
this can go in its own commit, since it is just good logging ideas, and not part of the VDSM specific changes.